src/effects/nyquist/Nyquist.cpp

```diff
@@ -1239,8 +1239,8 @@ bool NyquistEffect::ProcessOne()
          for (size_t i=0; i<ca.size(); i++) {
             if (i < 1000) {
                clips += wxString::Format(wxT("(list (float %s) (float %s))"),
-                                         Internat::ToString(ca[i]->GetStartTime()),
-                                         Internat::ToString(ca[i]->GetEndTime()));
+                                         Internat::ToString(ca[i]->GetPlayStartTime()),
```

Comment:
This change is correct, but discuss with Steve Daulton whether new information about the size of the hidden parts of the clip should also be passed to Nyquist.

Understand what happens here: C++ composes a string that is passed to the Lisp interpreter, and that sets up variables that are available to programming in Lisp.